src/tooling/piral-cli/src/apps/build-piral.ts

```diff
@@ -98,6 +100,9 @@ export async function buildPiral(baseDir = process.cwd(), options: BuildPiralOpt
     await removeDirectory(dest.outDir);
   }
 
+  const pckg = await readJson(root, 'package.json');
+  const { postPiralBuildDevelop, postPiralBuildRelease } = pckg.scripts;
```

Comment:
Not 100% happy with the names - I think something like `piral:post-emulator` and `piral:post-release` could make more sense (but then I would argue we potentially also need `piral:pre-emulator` and `piral:pre-release`, too - for symmetry).

Reply:
No strong opinion on this beside aiming for consistency.

`piral:postbuild-develop` and `piral:postbuild-release` would be my preference. `postbuild` to match [npm-scripts convention](https://docs.npmjs.com/misc/scripts)

`develop` instead of `emulator` because of the `--type develop` (and output dir name), though I don't mind if type `develop` has an `emulator` alias.

Reply:
OK sounds good to me. :+1:

Maybe we could also introduce `piral:postbuild`, i.e., to have something that *always* triggers independent of the `type` (so you can choose, do you always want that or only for a certain type).

Regarding the `type` - I don't know yet if we keep`develop` and `release`. I think having `emulator` and `release` could make more sense (for v1) to match the terminology we usually use (and to avoid confusing the develop with the debug mode for the Piral instance, which is usually meant to be the same thing).